doc/Testing.md

```diff
@@ -98,6 +126,60 @@ For more documentation on testing, read the references listed at the bottom.
     }
     ```
 
+## Functionnal/Integration Testing
+
+- We use [integration_test](https://docs.flutter.dev/testing/integration-tests) to create functionnal/integration tests. You can create a test like this :
+
+	```cs
+	// Import the test package
+	import 'package:flutter_test/flutter_test.dart';
+	import 'package:integration_test/integration_test.dart';
+
+	void main() {
+	testWidgets('Get Dad Jokes', (WidgetTester tester) async {
+		// Arrange
+		await tester.pumpWidget(const App());
+
+		// Without this the dadjokesContainer isn't there yet.
+		await tester.pumpAndSettle();
+
+		var dadjokesContainer = find.byKey(const Key('DadJokesContainer'));
+
+		// Act
```

Comment:
I'm not sure this is actually an **Act** because you're just fetching some widgets. It feels more like part of the **Assert**.
To me the Act of this test is simply the creation of the App.